apps/agent/app/src/script.js

```diff
@@ -47,290 +74,463 @@ const retryEvery = async (
   return attempt()
 }
 
-retryEvery(() => {
+retryEvery(() =>
+  // There are no settings to check, so we do a throwaway check to `initialized()`
+  // just to make sure we can connect properly
+  app
+    .call('hasInitialized')
+    .toPromise()
+    .then(() => initialize())
+).catch(_ => {
```

Comment:
There seems to be some weird indentation here

Reply:
Hmm, that's the linter doing it.